aspnetcore/client-side/angular/sample/AngularJSSample/src/AngularJSSample/Controllers/Api/PersonController.cs

```diff
@@ -16,7 +18,11 @@ public JsonResult GetPeople()
                 new Person { Id = 1, FirstName = "Bob", LastName = "Parker" }
             };
 
-            return Json(people);
+            // force the Json serializer to keep the case of the model properties
+            var serializerSettings = new JsonSerializerSettings();
+            serializerSettings.ContractResolver = new DefaultContractResolver();
+
```

Comment:
@Rick-Anderson I agree with your suggestion. The less code the reader has to look at, the easier the doc will be to follow. The comment will really help clarify.